File: src/main/java/no/stunor/origo/eventorapi/model/origo/start/TeamStart.java
package no.stunor.origo.eventorapi.model.origo.start;


import java.util.Date;
import java.util.List;

import no.stunor.origo.eventorapi.model.firestore.Organisation;

public record TeamStart(
    String startId,
    List<Organisation> organisations,
    List<TeamMemberStart> teamMembers,
    String teamName,
    Date startTime,
    String bib,
    String eventClassId) {
}

File: src/main/java/no/stunor/origo/eventorapi/model/event/startlist/TeamStart.kt
package no.stunor.origo.eventorapi.model.event.startlist

import com.google.cloud.Timestamp
import no.stunor.origo.eventorapi.model.organisation.Organisation

data class TeamStart(
        override var raceId: String = "",
        override var eventClassId: String = "",
        var organisations: List<Organisation> = listOf(),
        var teamMembers: List<TeamMemberStart> = listOf(),
        override var name: Any = "",
        override var bib: String? = null,
        override var startTime: Timestamp? = null,
) : CompetitorStart